static/app/views/settings/account/notifications/parentLabel.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import Avatar from 'app/components/avatar';
+import space from 'app/styles/space';
+import {OrganizationSummary, Project} from 'app/types';
+import {getParentKey} from 'app/views/settings/account/notifications/utils';
+
+type Props = {
+  notificationType: string;
+  parent: OrganizationSummary | Project;
+};
+
+/** TODO(mgaeta): Infer parentKey from parent. */
```

Comment:
did you mean to leave this in?

Reply:
Yeah I don't know enough about React today but someday I'll be able to simplify this function.